mappings/net/minecraft/block/RedstoneWireBlock.mapping

```diff
@@ -18,6 +18,8 @@ CLASS net/minecraft/unmapped/C_hbdlqish net/minecraft/block/RedstoneWireBlock
 	FIELD f_tfkeexfj SHAPES Ljava/util/Map;
 	FIELD f_vvlqyglw SHAPES_UP Ljava/util/Map;
 	FIELD f_vvxqbtel WEST I
+	METHOD m_ahfsctfo ([Lnet/minecraft/unmapped/C_vgpupfxx;)V
+		ARG 0 positions
```

Comment:
hm, i just realized that you can't really make a plural of `pos`; maybe `poses`, `posi`? :p
yeah, keep it as is